lib/builders/latexmk.js

```diff
@@ -77,12 +80,35 @@ export default class LatexmkBuilder extends Builder {
 
     if (engine.match(latexRegExp)) {
       args.push(`-latex="${engine}"`)
-      if (outputFormat !== 'dvi') {
-        atom.notifications.addWarning(
-          'PS/PDF output is not supported by LaTeX engines.'
-        )
+      if (outputFormat === 'pdf') {
+        switch (producer) {
+          case 'default':
+            // default is an option to protect a user's .latexmkrc from override
+            args.push('-pdfdvi')
+            break
+          case 'dvipdf':
+            args.push('-pdfdvi')
+            args.push('-e "\\$dvipdf = \'dvipdf %O %S %D\';"')
+            break
+          case 'dvipdfm':
```

Comment:
Just to make sure: I meant that `dvipdfm` is symlinked to `dvipdfmx`. In TeX Live 2016, `dvipdfmx` is symlinked to`xdvipdfmx`, as @yitzchak confirmed, and xdvipdfmx changes its behavior according to the extension of input file (`.dvi` or `.xdv`). That is not the case with TeX Live 2015 or earlier.